File: Backend/WebApi/Models/Game.cs
using System;
using System.Collections.Generic;

namespace WebApi.Models
{
    public class Game
    {
        public Guid Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public byte[] Poster { get; set; }
        public DateTime ReleaseDate { get; set; }
        public double Rating { get; set; }
        public int CountReviews { get; set; }
        public int GenreId { get; set; }
        public Genre Genre { get; set; }
        public ICollection<Comment> Comments { get; set; }
    }
}

File: Frontend/src/app/models/game.ts
import { SafeUrl } from "@angular/platform-browser";
import { Guid } from "guid-typescript";
import { GameComment } from "./comment";
import { Genre } from "./genre";

export class Game {
  constructor(
      public id: Guid,
      public name: string,
      public description: string,
      public poster: string,
      public posterUrl: SafeUrl,
      public releaseDate: string,
      public rating: number,
      public countReviews: number,
      public genreId: number,
      public genre: Genre,
      public comments: GameComment[]
    ) {}
}